migrations/__init__.py

```diff
@@ -24,7 +24,7 @@ def get_version(db: DuckDBPyConnection) -> int:
     try:
         return int(db.execute("SELECT MAX(id) FROM migrations").fetchone()[0])
     except:
-        return 0
+        return 1
```

Comment:
get_version() is supposed to return the last migration applied to the database, and if none has been applied, it only makes sense that this is lower than the first actual migration...?

Reply:
Ah, yes. That makes more sense! 😅